src/WpCli/eightshift-forms/assets/form.js

```diff
@@ -0,0 +1,18 @@
+export class Form {
+	constructor(options) {
+		this.formSelector = options.formSelector;
+
+		this.CLASS_ACTIVE = 'is-active';
+	}
+
+	// Init all actions.
+	init = () => {
+		const elements = document.querySelectorAll(this.formSelector);
+
+		if (elements.length) {
+			[...elements].forEach(() => {
+
+			});
```

Comment:
This is empty on purpose?

Reply:
yes. this is just a head start for your project not and actual usage in the forms. Think of this as a example :D